programs/perpetuals/src/instructions/add_liquidity.rs

```diff
@@ -200,7 +200,11 @@ pub fn add_liquidity(ctx: Context<AddLiquidity>, params: &AddLiquidityParams) ->
 
     // update pool stats
     msg!("Update pool stats");
-    pool.aum_usd = pool_amount_usd;
+    pool.aum_usd = pool_amount_usd.wrapping_add(
+        token_price
+            .get_asset_amount_usd(params.amount, custody.decimals)?
```

Comment:
good catch! it just should be deposit_amount here

Reply:
also lets use checked_add here

Reply:
you correctly compute the usd value from token amount with get_asset_amount_usd()
I just said you should be using `deposit_amount` instead of `params.amount`, because not entire amount goes into the pool, part is taken away as protocol fees. Same thing in remove_liquidity.rs
You can see which variable is used to adjust custody.assets.owned and use the same to adjust pool_aum_usd (after converting to usd of course)